OpenDreamRuntime/DreamRuntime.cs

```diff
@@ -123,6 +123,7 @@ public class DreamRuntime
 
             Map = new DreamMap(this);
             Map.LoadMap(CompiledJson.Maps[0]);
+            WorldInstance.SetVariable("maxz", new DreamValue(Map.Levels.Count));
```

Comment:
Why is this needed?

Reply:
...To set the value of the variable?

Reply:
It shouldn't need to be set, `DreamMetaObjectWorld`'s `OnVariableGet` handles `maxz`

Reply:
I completely forgot the special variable handlers were a thing 🤦‍♂️